### PR TITLE
USDt exit isolation mode - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20230926_AaveV3_Ava_EnablingUSDTAsCollateralOnAaveV3AVAXMarket/AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926.sol [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * @title Enabling USDT as collateral on Aave v3 AVAX Market
+ * @author Marc Zeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x5623b5f84f021ad787033b4a1efd9e2de417004d27c5f2e3d7351f9b575574b1
+ * - Discussion: https://governance.aave.com/t/arfc-enabling-usdt-as-collateral-on-aave-v3-avax-market/14632/3
+ */
+contract AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926 is
+  AaveV3PayloadAvalanche
+{
+  uint256 public constant BORROW_CAP = 80_000_000; // 80M
+  uint256 public constant SUPPLY_CAP = 100_000_000; // 100M
```
Comment (kyzia551) [inline src/20230926_AaveV3_Ava_EnablingUSDTAsCollateralOnAaveV3AVAXMarket/AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926.sol:17]: Supply cup is wrong, on forum and in readme is 170M

**Review by kyzia551 — CHANGES_REQUESTED** [review]

File: src/20230926_AaveV3_Ava_EnablingUSDTAsCollateralOnAaveV3AVAXMarket/AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926.sol (hunk 1 above)
Comment (marczeller) [reply src/20230926_AaveV3_Ava_EnablingUSDTAsCollateralOnAaveV3AVAXMarket/AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926.sol:17]: markdown has been updated, proposed caps are in line with risk service providers recommendations

**Review by kyzia551 — APPROVED** [review]

**Comment by Zer0dot** [comment]
<details>
  <summary>Review</summary>

### Code Review
- [x]  Dependencies should be imported from “official” repositories if possible/applicable (from Aave Github and BGD).
- [x]  A test suite exists and minimally checks the main aspects of the payload.
- [x]  Comments are consistent with the code.
- [x]  References to Snapshot/governance forum are correct.
- [x]  The proposal payload only has constants, and no storage variable (critically dangerous as it would mess with the Executor state)
- [x] Does **not** revoke permissions from itself (was a pattern in stewards, but critically dangerous in new delegatecall approach)
- [x]  The Foundry/Hardhat script to create the proposal is correct.
    - [ ]  If its crosschain its using correct forwarder / signature
- [x]  Addresses are correct
    - [ ]  Price feeds are correct. (Should be checked that the correct feed (usd/eth) is used for the specific AIP use case
- [x]  The AIP is not affecting / affected by other in progress AIPs

### Test Review
- [x]  All addresses are used in tests (including tokens / feeds etc) (ex: using incorrect feed for token (usd instead of eth) for token exchange)
- [x]  All use cases are tested

### AIP Review
- [x]  Adhering to the basic AIP template: header format and sections.
- [x]  If there are images, they are properly added to the AIP repository, as described on its README.
- [x]  The target branch for the PR is correct, as described in the README.
- [x]  The folder location of the AIP is correct.
- [x]  The file naming of the AIP is correct.
- [x]  Has payload implementation link (Github or if not open-sourced, Etherscan).
- [x]  Has link to tests (only applicable if open-sourced).

</details>

**Comment by Zer0dot** [comment]
<details>
  <summary>Review</summary>

### Code Review
- [x]  Dependencies should be imported from “official” repositories if possible/applicable (from Aave Github and BGD).
- [x]  A test suite exists and minimally checks the main aspects of the payload.
- [x]  Comments are consistent with the code.
- [x]  References to Snapshot/governance forum are correct.
- [x]  The proposal payload only has constants, and no storage variable (critically dangerous as it would mess with the Executor state)
- [x] Does **not** revoke permissions from itself (was a pattern in stewards, but critically dangerous in new delegatecall approach)
- [x]  The Foundry/Hardhat script to create the proposal is correct.
    - [ ]  If its crosschain its using correct forwarder / signature
- [x]  Addresses are correct
    - [ ]  Price feeds are correct. (Should be checked that the correct feed (usd/eth) is used for the specific AIP use case
- [x]  The AIP is not affecting / affected by other in progress AIPs

### Test Review
- [x]  All addresses are used in tests (including tokens / feeds etc) (ex: using incorrect feed for token (usd instead of eth) for token exchange)
- [x]  All use cases are tested

### AIP Review
- [x]  Adhering to the basic AIP template: header format and sections.
- [x]  If there are images, they are properly added to the AIP repository, as described on its README.
- [x]  The target branch for the PR is correct, as described in the README.
- [x]  The folder location of the AIP is correct.
- [x]  The file naming of the AIP is correct.
- [x]  Has payload implementation link (Github or if not open-sourced, Etherscan).
- [x]  Has link to tests (only applicable if open-sourced).

</details>

**Review by Zer0dot — APPROVED** [review]

**Pull request merged**